gossip/src/cluster_info.rs

```diff
@@ -1173,11 +1171,11 @@ impl ClusterInfo {
             .time_gossip_read_lock("get_votes", &self.stats.get_votes)
             .get_votes(cursor)
             .map(|vote| {
-                let transaction = match &vote.value.data {
-                    CrdsData::Vote(_, vote) => vote.transaction().clone(),
-                    _ => panic!("this should not happen!"),
+                let label = vote.value.label();
+                let CrdsData::Vote(_, vote) = &vote.value.data else {
+                    panic!("this should not happen!");
                 };
-                (vote.value.label(), transaction)
+                (label, vote.transaction().clone())
```

Comment:
could be more uniform to replace `label` in `(label, vote.transaction().clone())` with `(vote.value.label(), vote.transaction().clone())`, would match the `vote.transaction().clone()`. since that wasn't written as `transaction = vote.transaction().clone()` and then `(label, transaction)`

Reply:
basically it may be more uniform to write this as either:
```(label, transaction)``` 
OR
```(vote.value.label(), vote.transaction.clone())```

Reply:
```rust
let label = vote.value.label();
```
is needed because the initial `vote` is shadowed by `CrdsData::Vote(_, vote)`.